[WIP] Gave some love to SolarSystemNameGenerator

airxhi opened the pull request:
Added some basic functionality to SolarSystemNameGenerator.java which still needs some touching up.

airxhi committed:
Gave some love to SolarSystemNameGenerator

Added some basic functionality to SolarSystemNameGenerator.java which still needs some touching up.

Brutus5000 commented:
This looks nice. Is this finished or work in progress?
Would love to see the `solar system #x` dying too :)

airxhi commented:
Still some work to do, was just a quick half-hour mock up between classes. Need to do some research on some other naming systems to get some more variety and tweak some of the distributions.

airxhi committed:
Completed a passable SolarSystemNameGenerator

Added a couple of extra options to the constellations and added a list of other prefixes (gathered from various surveys). Tidied up the next() function and refactored the name generation into separate function and added generateNames(int amount) and clear() which returns a given amount of star system names and clears the current set of names respectively.

airxhi committed:
Fixed a spelling error

Managed to forget to commit one of the line changes, fixed now

Brutus5000 commented:
@Airxhi Thank you for this PR. If you have any more improvements, just open a new one.
